examples/denner-articles/advertised-article-update.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Denner\Client\ArticlesClient;
+
+$config = require realpath(__DIR__ . '/../bootstrap.php');
+
+$advertisedArticleId = @$_GET['advertised_article_id'] ?: '44444444-aaaa-4444-aaaa-444444444444';
+
+if (!$advertisedArticleId) {
+    throw new RuntimeException('Missing or invalid parameter "advertised_article_id"');
+}
+
+$params = array(
+    'advertised_article_id' => $advertisedArticleId,
+    'remarks' => 'Das ist ein Test',
```

Comment:
Please choose a different example (remarks is not really intended for changing through the API). One of the following would be better: promoted_override, instead_prices, manuscript_notes